packages/account/src/Sections/Security/SelfExclusion/self-exclusion.jsx

```diff
@@ -259,6 +244,7 @@ class SelfExclusion extends React.Component {
                 setSubmitting(false);
                 this.setState({ show_confirm: false, is_loading: true, is_confirm_page: false });
                 this.getSelfExclusion();
+                this.getLimits();
```

Comment:
can we call `getLimits` function in `client-store` instead?

Reply:
Do you mean to call it in `client-store` as well, or instead of this call? Because I believe there was some asynchronous issue when we call it in client store only.

Reply:
yes, instead of this call
`getLimits` in client store is cached in storage so that we don't make multiple requests of the same type. Also, it's better to send WS requests from a store

Reply:
Ok, If so, we can remove getSelfExclusion from here as well, since it's in client store

Reply:
I removed getLimits API call from here